test/docs/OpenJ9TestUserGuide.md

```diff
@@ -241,6 +241,17 @@ There are 3 ways to add options to your test run:
 
 - If you want to change test options, you can update playlist.xml in the corresponding test project.
 
+#### Run test or group of tests multiple times
+- Using TEST_ITERATIONS to specify the number of iterations. The test (for group target, each test in the group) will be executed multiple times. The test will pass if all the iterations are passed.
```

Comment:
Only one suggestion, otherwise looks good and thanks for adding this! After executed multiple times, yay be worth saying "This feature is designed to repetitively run small test targets, use with caution for top-level test targets that may take a long time to run".  or some such thing and you could pick a smaller test target for the example then.

Reply:
Thanks, I have updated the commit.